provider/pkg/provider/provider.go

```diff
@@ -220,7 +221,7 @@ func (p *googleCloudProvider) Diff(_ context.Context, req *rpc.DiffRequest) (*rp
 		}
 	}
 
-	return &rpc.DiffResponse{Changes: rpc.DiffResponse_DIFF_UNKNOWN, Replaces: replaces}, nil
+	return &rpc.DiffResponse{Changes: rpc.DiffResponse_DIFF_UNKNOWN, Replaces: replaces, DeleteBeforeReplace: true}, nil
```

Comment:
Why is this change needed? Isn't this a major behavior change?

Reply:
Ah, I should have commented on that. It's unrelated to this PR, I just noticed that replaces are failing while I was testing. We don't have auto-naming yet, so any resource needs to be deleted first to free up its name.